v3/newrelic/internal_app.go

```diff
@@ -134,13 +134,21 @@ func (app *app) doHarvest(h *harvest, harvestStart time.Time, run *appRun) {
 
 func obfuscateLicenseKeyCollectorRequest(responseErrorString string) string {
 	licenseKeyIndex := strings.Index(responseErrorString, "license_key=")
```

Comment:
It might be better to look for "&license_key=" or "?license_key=" in case "license_key" happens to be text that appears in one of the other data fields in the string.